ouroboros-consensus/src/consensus-testlib/Test/Ouroboros/Consensus/ChainGenerator/RaceIterator.hs

```diff
@@ -13,6 +13,22 @@ module Test.Ouroboros.Consensus.ChainGenerator.RaceIterator (
   , nextConservative
   ) where
 
+{- | These functions iteratively produce all race windows in a slot vector.
+
+The first window is produced by 'init', which unconditionally starts the window at the first slot.
+This window can then be passed to 'next', which starts the new window after the first active slot.
+
+@
+---X--X--X--X
+^ start of window 1 from 'init'
+    ^ start of window 2 from 'next'
+       ^ start of window 3 from 'next'
+@
```

Comment:
Maybe explicitly state `k=1` for this example (otherwise, window 3 can not be returned by `next` as it contains only two active slots), or extend the schema with dots to indicate that it does not necessarily end after the fourth active slot?

Reply:
✅